f2/src/forms/defaultValues.js

```diff
@@ -66,7 +66,7 @@ const formDefaults = {
     fileDescriptions: [],
   },
   location: { postalCode: '', city: '', province: '' },
-  contactInfo: { fullName: '', email: '', phone: '' },
+  contactInfo: { emailORphone: '', fullName: '', email: '', phone: '' },
```

Comment:
Why do we need to alter the default values? We are not adding a value to the report.

Reply:
Removed